docs/contributing/3.0-development/plugin-layout.rst

```diff
@@ -0,0 +1,86 @@
+Pulp Plugin Application Layout
+==============================
+
+This documentation is for Pulp Plugin developers. For Platform development,
+see the :doc:`app-layout`.
+
+The Pulp Platform does not manage any content itself. This functionality is provided by
+its plugins, which use the Pulp Platform Plugin API to manage specific types of content,
```

Comment:
iirc this is the only place it's listed so verbosely, which is intentionally done at the top of the document introducing this idea.